integration-tests/testsetups/keeper_benchmark.go

```diff
@@ -240,8 +240,9 @@ func (k *KeeperBenchmarkTest) Run() {
 			txKeyId = 0
 		}
 		kr := k.keeperRegistries[rIndex]
+		// TODO: need to add the LINK, WETH and WETH/USD feed to support v23
```

Comment:
is this not done?
if so, can u create a ticket to track

Reply:
Yes, this is not done yet, Sync-ed with Anirudh this morning, and he is aware and gonna tackle this soon. Here is the ticket (AUTO-11119) for benchmark testing.